drivechain-server/bdk/bdk.go

```diff
@@ -18,6 +21,9 @@ import (
 	"github.com/tidwall/gjson"
 )
 
+//go:embed bin/bdk-cli
```

Comment:
This fails at build time because the file isn't there. It would be nice if it would be possible to build the server binary, _without_ building bdk-cli. Could be solved by doing a wildcard embed, I believe. `go:embed bin/*` should to the trick. Might also need to add an empty `bin/.gitkeep` file that's checked into Git. Believe embedding wildcards fail if there's no matches

Reply:
Good point!